terraform/modules/bucket/main.tf

```diff
@@ -13,6 +13,13 @@ resource "aws_s3_bucket" "this" {
   force_destroy = true
 }
 
+resource "aws_ssm_parameter" "bucket" {
+  count = var.ssm_parameter != null ? 1 : 0
+  name  = var.ssm_parameter
+  value = aws_s3_bucket.this.id
+  type  = "String"
+}
+
```

Comment:
Let's see if we can drop the creation of the ssm_parameter. The lambdas can reference the buckets directly by name. It strikes me as an issue that is a bit annoying at bucket creation, but once we have the bucket name it's done.

Reply:
Can they update to reference the bucket name directly?